CLIENT-SPECIFICATION.md

```diff
@@ -240,11 +241,11 @@ the commit hash changes when merging with squash or rebase.
 
  - [v1.3, June 11th 2020](https://github.com/tldr-pages/tldr/blob/v1.3/CLIENT-SPECIFICATION.md) ([#4101](https://github.com/tldr-pages/tldr/pull/4101))
    - Clarified fallback to English in the language resolution algorithm.
-   - Update `LANG` and `LANGUAGE` environment variable to conform to the GNU spec.
+   - Update the `LANG` and `LANGUAGE` environment variables to conform to the GNU spec.
 
  - [v1.2, July 3rd 2019](https://github.com/tldr-pages/tldr/blob/v1.2/CLIENT-SPECIFICATION.md) ([#3168](https://github.com/tldr-pages/tldr/pull/3168))
    - Addition of a new `-L, --language` recommended command-line option.
-   - Rewording of the language section also encouraging the use of configuration files for language.
+   - Rewording of the language section also encourages the use of configuration files for language.
```

Comment:
I don't think this matches the original intent of the sentence. Maybe a comma helps here?


```suggestion
   - Rewording of the language section, also encouraging the use of configuration files for language.
```